recipes/zserio/all/conanfile.py

```diff
@@ -65,6 +86,8 @@ def build(self):
         cmake.build()
         sources = self.conan_data["sources"][self.version]
         get(self, **sources["compiler"], pattern="zserio.jar")
+        if Version(self.version) >= "2.14.0":
```

Comment:
It could be downloaded in source(). The compiler is different because is a binary, but cmake file we download directly to sources.

Reply:
I don't understand. The script is released as part of the [compiler zip release](https://github.com/ndsev/zserio/releases/download/v2.14.0/zserio-2.14.0-bin.zip) since it is an utility for compiler users -> to be able to call the compiler from any CMake project.

Should I have an extra source like [this](https://raw.githubusercontent.com/ndsev/zserio/v2.14.0/cmake/zserio_compiler.cmake)? And use it only from Zserio version 2.14.0 since it wasn't in the repository at all in Zserio 2.13.0?

Reply:
Okay, makes sense.